### PR TITLE
Compatible with Tensorflow 1.1.0rc1 + Python 3.5.2

### PR DESCRIPTION
**Pull request by lordet01**
Here is the patch files for compatible with TF 1.1.0 + Python 3.
In addition, I wrote a simple script to enhance multiple wave files in a folder.

I am also sharing the model that was trained from my PC with the modified codes.
https://drive.google.com/open?id=0BzoVVRWvLDl1ajNtTHdDaHpZT0E
https://drive.google.com/open?id=0BzoVVRWvLDl1T0R2NFBlMVV1VW8
https://drive.google.com/open?id=0BzoVVRWvLDl1a3hWWUc1Si1OeW8

Best regards,

-Kwang Myung Jeon

### CODE REVIEW

**Comment by rafaelvalle** [comment]
Please also use the code below to replace the clean method on the SEGAN class because it is extremely slow.

    def clean(self, x):
        """ clean a utterance x
            x: numpy array containing the normalized noisy waveform
        """
        # zero pad if necessary
        remainder = len(x) % (2 ** 14)
        if remainder != 0:
            x = np.pad(x, (0, 2**14 - remainder), 'constant', constant_values=0)
        # split files into equal 2 ** 14 sample chunks
        x = np.array(np.array_split(x, int(len(x) / 2 ** 14)))
        x_ = np.zeros((self.batch_size, 2 ** 14))
        x_[:len(x)] = x
        fdict = {self.gtruth_noisy[0]: x_}
        output = self.sess.run(self.Gs[0], feed_dict=fdict)[:len(x)]
        output = output.flatten()
        # remove zero padding if added
        if remainder != 0:
            output = output[:-(2**14 - remainder)]
        return output

**Review by asadullah797 — APPROVED** [review]

**Review by asadullah797 — APPROVED** [review]

**Review by asadullah797 — APPROVED** [review]
